### DISCUSSION TITLE
I think the sign up is broken

### DISCUSSION THREAD

**Discussion by arminvburren** [fief-dev/fief#377] [General]
On one of my app, i keep trying to register new accounts and it doesn't send any mail to validate.

Doesn't work either when you click on "i forgot my password"

Is this normal / related to the migration we're supposed to do ?

Regards,

Edit it might be since 14/05 actually since it's th elast time a user registered on another app, and I was having almost one every two days before that

**Comment by frankie567** [comment]
Fief Cloud is operating normally; however, it looks like emails delivery is blocked on Hotmail/Outlook mailboxes. I'll try to see if I can do something with our email provider.

In the meantime, you can try to configure your own email sending address (with proper DNS authentication for your domain): https://docs.fief.dev/configure/tenants/#domain-authentication This may help with deliverability.

---

Of course, regarding the migration, I still recommend you to do it as soon as possible.
